.pipelines/OneBranch.Release.yml

```diff
@@ -161,6 +162,7 @@ extends:
             dependsOn: 'publish_msal_common'
             publishOverride: ${{ parameters.overrideMsalNodeExtensions }}
             testMode: ${{ parameters.testMode }}
+            poolType: 'windows'
```

Comment:
one branch is only for windows?

Reply:
Extensions must be published using a windows machine since we are now prebuilding the windows binaries. All other libs will be published using an ubuntu machine